sqldb/migrations.go

```diff
@@ -14,10 +16,36 @@ import (
 	"github.com/golang-migrate/migrate/v4/source/httpfs"
 )
 
+// MigrationConfig is a configuration struct that is used to describe in-code
+// migration targets. Each such migration is applied at a specific schema
+// version.
+type MigrationConfig struct {
+	// Name is the name of the migration.
+	Name string
+
+	// Version is the schema version at which the migration is applied.
+	Version int
```

Comment:
"schema version" as in `up` file number yeah? if so, what about if we have 2 code-level migrations in a row  that depend on each-other/where ordering is important?

Reply:
hmm ok I see the order is gleaned implicitly from the order in which it is passed to `ApplyMigrations`

Reply:
With this latest update, we now have two versions: one for the schema, tracked by the golang-migrate files, and another managed by our migration tracker table, which records the 'global' database version.